com.unity.formats.fbx/Editor/FbxExporter.cs

```diff
@@ -1687,10 +1635,10 @@ private bool ExportParentConstraint(IConstraint uniConstraint, FbxScene fbxScene
 
                 fbxParentConstraint.AddConstraintSource(uniSource.node, uniSource.weight);
                     
-                var fbxTranslationOffset = ConvertToRightHanded(uniTranslationOffset, UnitScaleFactor);
+                var fbxTranslationOffset = ConvertToFbxVector4(uniTranslationOffset, UnitScaleFactor);
                 fbxParentConstraint.SetTranslationOffset(uniSource.node, fbxTranslationOffset);
                     
-                var fbxRotationOffset = ToFbxVector4(ConvertToRightHandedEuler(uniRotationOffset));
+                var fbxRotationOffset = ToFbxVector4(ToFbxDouble3(uniRotationOffset));
```

Comment:
why not `ConvertToFbxVector4`?

Reply:
(if there's a reason please document it, otherwise please use the same as all the others)